src/reversible_tokenize.jl

```diff
@@ -0,0 +1,54 @@
+# A simple reversible tokenizer
+
+MERGESYMBOL = '~'
+
+function is_weird(c::AbstractChar)
+    return .!(isletter(c) ||isnumeric(c) || isspace(c))
```

Comment:
```suggestion
    return !(isletter(c) ||isnumeric(c) || isspace(c))
```

the `.!` is broadcasting and this isn't breadcasting